staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller_test.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flowcontrol
+
+import (
+	"reflect"
+	"testing"
+
+	flowcontrol "k8s.io/api/flowcontrol/v1beta1"
+)
+
+func Test_configController_generatePatchBytes(t *testing.T) {
+	type args struct {
+		condition flowcontrol.FlowSchemaCondition
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    []byte
+		wantErr bool
+	}{
+		{
+			name: "check if only condition is parsed",
+			args: args{
+				condition: flowcontrol.FlowSchemaCondition{
```

Comment:
The normal code sets more fields of the FlowSchemaCondition; let's test that.

Reply:
I am sorry; I do not know how this API works; would you be able to point me to the either e2e's or examples that I can look from and understand.

Reply:
I think I understand what you are saying, I think you are asking me to check with all the fields in Condition. Please correct me if I am wrong.

Reply:
Added more test cases, let me know if I understood things properly.

Reply:
yes, you got it.
Thanks!